mackerel-plugin-windows-server-sessions/lib/sessions.go

```diff
@@ -43,9 +54,11 @@ func (m WindowsServerSessionsPlugin) FetchMetrics() (map[string]interface{}, err
 	if err != nil {
 		return nil, err
 	}
-	stat := make(map[string]interface{})
-	for k, v := range counts {
-		stat["windows.server.sessions."+k+".count"] = uint64(v)
+	stat := make(map[string]interface{}, len(counts))
+	for _, v := range counts {
+		// node name of Windows can contain ".", which is the metric name delimiter on Mackerel.
+		nodeMetricKey := strings.ReplaceAll(v.Node, ".", "_")
```

Comment:
In my opinion, keeping `.` in the metric name was a bug, so this does not break compatibility.

Reply:
Of course, I think it is ok to add `-legacy` option to keep behavior. But it is not required.

Reply:
Because there are some hosts with node name including ".", we added `-legacymetricname` option. 280503b